sdk/internal/recording/recording.go

```diff
@@ -624,11 +626,37 @@ func GetRecordingId(t *testing.T) string {
 }
 
 func GetRecordMode() string {
+	if !(recordMode == RecordingMode || recordMode == PlaybackMode || recordMode == LiveMode) {
+		recordMode = PlaybackMode
```

Comment:
What do you think of doing this in `init`? Then you get the desired default behavior and don't have to consider calling patterns around this package.

Reply:
Now that you've added `init()`, `recordMode` will have a value when this method executes